### PR TITLE
Cypress login refactoring

### PR DESCRIPTION
**Pull request by fullsushidev**
## **TLDR:**

- [x] Yet another login strategy
- [x] New fixture for login strategies
- [x] Simplifying commands and removing duplicated code
- [x] Some linters
<br>

## **Motivation:**
Currently, we have some flaky tests on pr_checks and some other when running locally.
This PR attempts to solve most of the login issues between environments by:

- Wrapping the login into a session to keep for all the specs and including a validation to re-login when the session expired, speeding up the tests in approx. ~60%.
- Cleaning the code by moving the login strategy values into a fixture, simplifying redundant commands and deleting duplicated code.
<br>

### **Testing:**
Expect 5 specs failing that will be fixed in a separate PR

- [x] Automation Calculator
- [x] Clusters
- [x] Global
- [x] Reports (flaky)
- [x] Reports Page

<br>

depends on [AA backend MR-771](https://gitlab.cee.redhat.com/automation-analytics/automation-analytics-backend/-/merge_requests/771)
related to [AA-1723](https://issues.redhat.com/browse/AA-1723)

### CODE REVIEW

**Review by skochay330 — APPROVED** [review]

**Comment by fullsushidev** [comment]
this work was moved to #988
